src/borg/archiver.py

```diff
@@ -1099,6 +1103,113 @@ def print_output(diff, path):
 
         return self.exit_code
 
+    @with_repository(compatibility=(Manifest.Operation.READ,))
+    def do_dump(self, args, repository, manifest, key):
```

Comment:
hmm, iirc there is some `borg debug dump` already. maybe a source of confusion if this is named the same?

Reply:
Indeed. Perhaps "save" is better ?